TypeCobol.LanguageServer/TypeCobolServer.cs

```diff
@@ -187,8 +168,13 @@ private void MissingCopiesDetected(object fileUri, MissingCopiesEvent missingCop
             //Warning the main file could not be opened
             //This event can be used when a dependency have not been loaded
 
+            // Get the original URI (which was set by the client)
+            // DON'T use ToString() as it returns the canonically unescaped form of the URI
+            // (it may cause issue if the path contains some blanks which need to be escaped)
+            string uri = ((Uri)fileUri).OriginalString;
```

Comment:
In my opininon, the previous implementation with the dedicated constructor was safer (because the usage of `URI.OriginalString` was centralized)
I think we should keep something similar.
If it's not possible to restore the constructor, maybe a builder `BuildFrom(URI)`

Reply:
Ok for the builder method